src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundUserAcceptanceTest.java

```diff
@@ -1,36 +1,37 @@
 package uk.nhs.digital.nhsconnect.lab.results.inbound;
 
+import lombok.extern.slf4j.Slf4j;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import uk.nhs.digital.nhsconnect.lab.results.IntegrationBaseTest;
 import uk.nhs.digital.nhsconnect.lab.results.mesh.message.OutboundMeshMessage;
 import uk.nhs.digital.nhsconnect.lab.results.mesh.message.WorkflowId;
-
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.Inbound;
+import uk.nhs.digital.nhsconnect.lab.results.utils.JmsHeaders;
 
 import javax.jms.JMSException;
 import javax.jms.Message;
+import java.io.IOException;
+import java.nio.file.Files;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 /**
  * The test EDIFACT message is sent to the MESH mailbox where the adaptor receives
  * inbound transactions. The test waits for the transaction to be processed and compares
- * the message from the inbound queue to be the same to the message which has been sent.
+ * the message from the GP Outbound queue to be the same to the message which has been sent.
```

Comment:
Is this accurate for the current state of the system? Ultimately the message on the Outbound GP queue will be a converted message, not the same as that which was originally sent, right?

Reply:
I've re-worded this slightly, hopefully it makes more sense now? This test will be more useful once we have the mapping in place, currently the outcome is an almost empty json file